schema/config.model.schema

```diff
@@ -442,7 +442,8 @@
           "required": true,
           "inputType": "Checkbox",
           "validators": [],
-          "title": "Use strict mode?"
+          "title": "Use strict mode?",
+          "help": "Uses ES6 modules. If AMD-style module definitions are required for plugins, turn this off"
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode

Not quite right. ES6 modules are used regardless of this switch.